### ISSUE TITLE
Signature not appearing on iPhone

### ISSUE THREAD

**Issue by divyang-dv** [thread-pond/signature-pad#223]
The Signature on an iPhone disappears as soon as you sign it. It works if you are using Chrome but doesn't work on Safari browser.

**Comment by lujain235** [comment]
Hi,
Do you solve this issue?

**Comment by mockdeep** [comment]
@lujain235 we haven't had any complaints from our users about this. Are you seeing issues? I'm not sure I'll have time to look into it, but if you want to make a PR I'll be happy to take a look.

**Comment by lujain235** [comment]
@mockdeep thanks for reply.
yes we are facing this issue on iPhone.
 Here is a video of the issue
 I thought it might be a resizing issue so i stopped the canvas resize code but the issue wasn't solved.
 Can you please take a look and advise.
 thanks.

https://user-images.githubusercontent.com/116438247/197758894-265678c6-4fdd-4742-ae55-9895bc0b22f1.mp4

**Comment by mockdeep** [comment]
@lujain235 I'm unfortunately not at all familiar with the code in this repo, so I can't offer much. I'll try to set aside some time to look into it some more and see if I can understand what is happening.

**Comment by lujain235** [comment]
Hi @mockdeep 
The issue has been resolved by setting drawBezierCurves =  false.

**Comment by mockdeep** [comment]
@lujain235 you must not be using this library. It [doesn't have any references to `drawBezierCurves`](https://github.com/thread-pond/signature-pad/search?q=drawBezierCurves).

**Comment by donkitchen** [comment]
> Hi @mockdeep The issue has been resolved by setting drawBezierCurves = false.

Where did you set this?  I'm having the same issues in our app.

**Comment by mockdeep** [comment]
@donkitchen @lujain235 where are you pulling the signature pad library from?

**Comment by lujain235** [comment]
@donkitchen @mockdeep Sorry for the conflicts, it turns out that our developers used another code for Bezier Curves and merged it with this library, and that was the cause of the issue, when i stopped it the library works. (when i added my comment i didn't know that this code had been added to the library by our team)

**Comment by donkitchen** [comment]
> @donkitchen @lujain235 where are you pulling the signature pad library from?

I downloaded the js library from here.  I am not, however, using the styles that you guys have for the examples.

Since the example linked in the repo works on my device, I suspect it may be a CSS/style thing, which I am going to verify today.  I updated my code to use the latest JS library and that didn't make a difference, so the styles are my next guess.

I don't really want to focus on the ones that change the look, more anything that may be functional, which is why I'm going to try the .pad class first.

**Comment by donkitchen** [comment]
I believe the problem seems to be my use case of having two signature elements on the same page.  I didn't see a demo of that so I can't know if that actually does still work now or not.

I've been trying to troubleshoot by making each class unique and passing those element names in as options, but not having any luck yet.

Is anybody else using two signatures on one page?  If they are, is it still working on IOS16?

**Comment by mockdeep** [comment]
We display a lot of signatures on the same page and haven't had any reports of it not working for people on IOS. Are you able to provide a code sample the reproduces the issue?

**Comment by donkitchen** [comment]
Yes, I'll work on that -- if you are willing, it may be easier for me to give you instructions on how you can access my stuff, with a brief video on how you would test it.  Since it's all HTML, CSS and JS you would easily be able to see all of the pieces working together.

**Comment by mockdeep** [comment]
Sure, that works too.

**Comment by donkitchen** [comment]
> Sure, that works too.

Sent you an email @mockdeep
